cms/templates/js/metadata-editor.underscore

```diff
@@ -3,4 +3,10 @@
     <li class="field comp-setting-entry metadata_entry">
     </li>
     <% }) %>
+    <li class="field comp-setting-entry metadata_entry">
+        <div class="wrapper-comp-setting">
+            <label class="label setting-label"><%= gettext("Location id") %></label>
```

Comment:
Should be using <%- instead of <%= (both this line and the next) to ensure the text is escaped.